docs/log-drains.md

```diff
@@ -0,0 +1,55 @@
+
+### Logs Drains
```

Comment:
Do we agree that this page is no longer linked on the main page? If I remember correctly the only way to activate the logs drain is threw the CLI, no?

Reply:
As far as I know, this doc should be rewrote here and in Clever Doc. I'll link it back when I'll be sure the content is OK. The goal was first to get a cleaner README and the iterate on this quickly